CONTRIBUTING.rst

```diff
@@ -259,10 +259,17 @@ Naming Conventions
 Guideline to write an example DAG
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - The example DAG should be self-sufficient as it is tested as part of the CI. For example, while implementing example DAG for ``S3KeySensorAsync``, the DAG should first create bucket, then upload s3 key, the check for key using ``S3KeySensorAsync`` and then finally delete the bucket once sensor found the key.
+- Read resource name like cluster, db, file name from environment variable avoid hard-coding it.
+- Read Airflow connection-id from environment variable and pass the value in respective task. We recommend to not depend on default connection-id.
+- Set the DAG ``schedule_interval`` parameter to ``None`` to avoid unnecessarily dag run since we trigger DAG externally.
+- Set the DAG ``catchup`` parameter to ``False``. This will avoid backfilling.
+- Add appropriate tags for the DAG. This will help to search DAG quickly.
 - Add proper doc-strings as part of example DAG.
 - Include a long running query always in the example DAG.
 - Include a clean up step at the start of the example DAG so that there won't be failures if the resources are already present.
+- Set ``trigger_rule`` parameter to ``all_done`` for cleanup task.
```

Comment:
```suggestion
- Set ``trigger_rule`` parameter to ``all_done`` for the cleanup task so that clean up happens irrespective of upstream task failures.
```